### PR TITLE
Add support for new CollectionGroup query

### PR DESCRIPTION
**Pull request by matthewtsmith**
Update firebase-admin to 8.2.0 (latest)
Update firestore to 2.2.1 (latest)
Add support for CollectionGroup

### CODE REVIEW

File: lib/src/firestore.dart [hunk 1]
```diff
@@ -76,6 +76,12 @@ class Firestore {
     return new CollectionReference(nativeInstance.collection(path), this);
   }
 
+  /// Gets a [DocumentQuery] for the specified Firestore path.
```
Comment (pulyaevskiy) [inline lib/src/firestore.dart]: This doc seems inaccurate. Maybe use the one from the JS binding?

**Comment by pulyaevskiy** [comment]
Also, wondering if you tested this with latest dependencies? It's pretty big version bump for firestore library...

**Comment by matthewtsmith** [comment]
@pulyaevskiy I looked over the release notes between 6.2.0 and 8.2.0 of the admin lib. As far as the API goes, I didn't see any breaking changes. The big change in 8.0 was to drop node 6 support.

I've been using my fork with this change for the last couple of weeks and have not found any issues so far.

File: lib/src/firestore.dart (hunk 1 above)
Comment (matthewtsmith) [reply lib/src/firestore.dart]: Doc comment updated.

**Comment by pulyaevskiy** [comment]
Hey, sorry for the delay here. Will take another look ASAP.

**Comment by matthewtsmith** [comment]
@pulyaevskiy Actually you may want to hold off. I'm in the process of adding `Messaging` support. I can close this PR and re-submit once I'm done with that as well.

**Comment by pulyaevskiy** [comment]
Sounds good, thanks for letting me know!

**Pull request closed without merging**